etc/config.xml

```diff
@@ -24,6 +24,7 @@
                 <can_use_checkout>1</can_use_checkout>
                 <can_authorize>1</can_authorize>
                 <can_capture>1</can_capture>
+                <can_refund_partial_per_invoice>1</can_refund_partial_per_invoice>
```

Comment:
have you confirmed that all these payment methods allow partial refunds? because not all our payment methods do

Reply:
The reason for adding this to all payment methods even though the payment method does not support it, is to allow merchants to change the order quantity while creating credit memo, which is used to refund order. Right now, merchants cannot change the order quantity.

For example, if a user orders 10 items but later want to refund 5 items then our plugin do not support that. Merchant would want to create a credit memo for 5 items only. But, our plugin will not allow that because the payment method do not have this partial refund config. By adding this config, we allow merchants to change the quantity so they can do partial refund

If a payment method do not support partial refund then we will receive the error from the API and show a message to the merchants telling them that the payment method do not support partial refund.

But by not adding for some payment methods, merchant won’t be able to change the quantity and might complain that s/he is unable to do partial refund, not knowing that the selected payment method do not support it.

Reply:
awesome, thanks for explaining @aashishgurung. It makes sense now.